wheel/src/api.rs

```diff
@@ -39,10 +42,40 @@ pub fn run_serialized_chia_program(
     adapt_response(py, allocator, r)
 }
 
+fn tuple_for_parsed_triple(py: Python<'_>, p: &ParsedTriple) -> PyObject {
+    let tuple = match p {
+        ParsedTriple::Atom {
+            start,
+            end,
+            atom_offset,
+        } => PyTuple::new(py, [*start, *end, *atom_offset as u64]),
+        ParsedTriple::Pair {
+            start,
+            end,
+            right_index,
+        } => PyTuple::new(py, [*start, *end, *right_index as u64]),
```

Comment:
I was thinking of maybe some implementing deserialization to `Allocator` using this, where we just copy the buffer into the allocator's memory and then use this array to populate the allocator elements. It would reduce the amount of code because we could delete the complicated allocator deserialization code. Probably not really worth it at this time though.